CMakeLists.txt

```diff
@@ -650,9 +650,7 @@ endif()
 add_subdirectory(Externals/cpp-optparse)
 
 find_package(fmt 8)
-if(fmt_FOUND)
-  message(STATUS "Using shared fmt ${fmt_VERSION}")
-else()
+if(NOT fmt_FOUND)
```

Comment:
Does `find_package(fmt 8 QUIET)` not work for this one?

Reply:
It does, but is very noisy if it doesn't find it, and this is more robust.

Reply:
What do you mean by robust?

Reply:
Able to find more types of installations